Jakefile.js

```diff
@@ -4,7 +4,6 @@ var fs = require("fs");
 var os = require("os");
 var path = require("path");
 var child_process = require("child_process");
-var Linter = require("tslint");

```

Comment:
we should think about switching Travis over to gulp so we don't have to duplicate effort like this in the future.